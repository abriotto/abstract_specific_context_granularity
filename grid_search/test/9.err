  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:01, 73.06it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:00<00:01, 67.28it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:00<00:01, 61.81it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:00<00:00, 67.28it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:00<00:00, 68.08it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:00<00:00, 65.10it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:00<00:00, 65.29it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:00<00:00, 62.00it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:01<00:00, 65.23it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:01<00:00, 67.35it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:01<00:00, 63.37it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:01<00:00, 68.70it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 66.19it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:00, 81.83it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:00<00:00, 70.08it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 66.26it/s]
