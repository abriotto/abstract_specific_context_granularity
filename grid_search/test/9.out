# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.702488124370575, "acc": 0.5143354535102844, "length": 3.432681083679199, "mode": "train", "epoch": 1}
{"loss": 0.6949154138565063, "acc": 0.5248618721961975, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6938411593437195, "acc": 0.5219189524650574, "length": 2.1670162677764893, "mode": "train", "epoch": 2}
{"loss": 0.6939236521720886, "acc": 0.511049747467041, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6688292026519775, "acc": 0.5975574851036072, "length": 1.847837209701538, "mode": "train", "epoch": 3}
{"loss": 0.659608006477356, "acc": 0.5862560868263245, "length": 1.5469613075256348, "mode": "test", "epoch": 3}
{"loss": 0.6261314749717712, "acc": 0.6559126377105713, "length": 1.9982378482818604, "mode": "train", "epoch": 4}
{"loss": 0.6202616095542908, "acc": 0.638888955116272, "length": 2.1602210998535156, "mode": "test", "epoch": 4}
{"loss": 0.6000248193740845, "acc": 0.6673290729522705, "length": 2.2177703380584717, "mode": "train", "epoch": 5}
{"loss": 0.6189467906951904, "acc": 0.6445796489715576, "length": 2.044198989868164, "mode": "test", "epoch": 5}
{"loss": 0.5912604928016663, "acc": 0.6864743828773499, "length": 2.1546216011047363, "mode": "train", "epoch": 6}
{"loss": 0.620087742805481, "acc": 0.6570886373519897, "length": 2.077347993850708, "mode": "test", "epoch": 6}
{"loss": 0.5768414735794067, "acc": 0.7003709077835083, "length": 2.242905855178833, "mode": "train", "epoch": 7}
{"loss": 0.6113243103027344, "acc": 0.6510293483734131, "length": 1.911602258682251, "mode": "test", "epoch": 7}
{"loss": 0.5596452951431274, "acc": 0.7157286405563354, "length": 2.266468048095703, "mode": "train", "epoch": 8}
{"loss": 0.5987311601638794, "acc": 0.6703115105628967, "length": 2.1933701038360596, "mode": "test", "epoch": 8}
{"loss": 0.5502434968948364, "acc": 0.7174564599990845, "length": 2.4064807891845703, "mode": "train", "epoch": 9}
{"loss": 0.5861204862594604, "acc": 0.6932249069213867, "length": 2.2928175926208496, "mode": "test", "epoch": 9}
{"loss": 0.5348789095878601, "acc": 0.7268359065055847, "length": 2.5200042724609375, "mode": "train", "epoch": 10}
{"loss": 0.5642415881156921, "acc": 0.6911134123802185, "length": 2.309392213821411, "mode": "test", "epoch": 10}
{"loss": 0.5174606442451477, "acc": 0.7481784820556641, "length": 2.671776294708252, "mode": "train", "epoch": 11}
{"loss": 0.534591019153595, "acc": 0.7375882863998413, "length": 2.6408839225769043, "mode": "test", "epoch": 11}
{"loss": 0.4884006977081299, "acc": 0.7773149609565735, "length": 2.674743890762329, "mode": "train", "epoch": 12}
{"loss": 0.5008994936943054, "acc": 0.7744508981704712, "length": 2.9060773849487305, "mode": "test", "epoch": 12}
{"loss": 0.4438261091709137, "acc": 0.8136570453643799, "length": 3.002927541732788, "mode": "train", "epoch": 13}
{"loss": 0.47396692633628845, "acc": 0.7890577912330627, "length": 3.287292718887329, "mode": "test", "epoch": 13}
{"loss": 0.4149151146411896, "acc": 0.8226621747016907, "length": 3.3074467182159424, "mode": "train", "epoch": 14}
{"loss": 0.4414612054824829, "acc": 0.8107283115386963, "length": 3.3701658248901367, "mode": "test", "epoch": 14}
{"loss": 0.3915553092956543, "acc": 0.8337188959121704, "length": 3.394625425338745, "mode": "train", "epoch": 15}
{"loss": 0.4330519139766693, "acc": 0.8134688138961792, "length": 3.5856354236602783, "mode": "test", "epoch": 15}
{"loss": 0.36136651039123535, "acc": 0.8444847464561462, "length": 3.550416946411133, "mode": "train", "epoch": 16}
{"loss": 0.43354710936546326, "acc": 0.8099788427352905, "length": 3.6519336700439453, "mode": "test", "epoch": 16}
{"loss": 0.35150107741355896, "acc": 0.856408417224884, "length": 3.5634360313415527, "mode": "train", "epoch": 17}
{"loss": 0.4264073073863983, "acc": 0.8235524892807007, "length": 3.668508291244507, "mode": "test", "epoch": 17}
{"loss": 0.3359396755695343, "acc": 0.8569703698158264, "length": 3.671872615814209, "mode": "train", "epoch": 18}
{"loss": 0.3906891942024231, "acc": 0.8350598812103271, "length": 3.6850829124450684, "mode": "test", "epoch": 18}
{"loss": 0.3273848593235016, "acc": 0.8630841970443726, "length": 3.7306556701660156, "mode": "train", "epoch": 19}
{"loss": 0.40307188034057617, "acc": 0.8306695818901062, "length": 3.7845304012298584, "mode": "test", "epoch": 19}
{"loss": 0.3188166618347168, "acc": 0.8671383857727051, "length": 3.7939906120300293, "mode": "train", "epoch": 20}
{"loss": 0.38733240962028503, "acc": 0.8322275876998901, "length": 3.80110502243042, "mode": "test", "epoch": 20}
{"loss": 0.30684980750083923, "acc": 0.8738027215003967, "length": 3.7963132858276367, "mode": "train", "epoch": 21}
{"loss": 0.35741597414016724, "acc": 0.8600327968597412, "length": 3.867403268814087, "mode": "test", "epoch": 21}
{"loss": 0.2945876121520996, "acc": 0.87794429063797, "length": 3.8863513469696045, "mode": "train", "epoch": 22}
{"loss": 0.37260445952415466, "acc": 0.8519496321678162, "length": 3.834254026412964, "mode": "test", "epoch": 22}
{"loss": 0.29365089535713196, "acc": 0.8793913722038269, "length": 3.8073484897613525, "mode": "train", "epoch": 23}
{"loss": 0.3638436496257782, "acc": 0.8550299406051636, "length": 3.9834253787994385, "mode": "test", "epoch": 23}
{"loss": 0.27011820673942566, "acc": 0.8949410319328308, "length": 3.935581684112549, "mode": "train", "epoch": 24}
{"loss": 0.36538442969322205, "acc": 0.8544832468032837, "length": 3.9944751262664795, "mode": "test", "epoch": 24}
{"loss": 0.26981687545776367, "acc": 0.8902838230133057, "length": 3.9387590885162354, "mode": "train", "epoch": 25}
{"loss": 0.34535011649131775, "acc": 0.8710867762565613, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.25868675112724304, "acc": 0.9013906717300415, "length": 3.9482486248016357, "mode": "train", "epoch": 26}
{"loss": 0.35446345806121826, "acc": 0.8609578609466553, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.2643972337245941, "acc": 0.8928138017654419, "length": 3.950181245803833, "mode": "train", "epoch": 27}
{"loss": 0.344823956489563, "acc": 0.8603931069374084, "length": 3.9834253787994385, "mode": "test", "epoch": 27}
{"loss": 0.2657531797885895, "acc": 0.8994085192680359, "length": 3.8783082962036133, "mode": "train", "epoch": 28}
{"loss": 0.38497287034988403, "acc": 0.8568249344825745, "length": 3.9060773849487305, "mode": "test", "epoch": 28}
{"loss": 0.25880101323127747, "acc": 0.898647665977478, "length": 3.933847427368164, "mode": "train", "epoch": 29}
{"loss": 0.3120152950286865, "acc": 0.8775323629379272, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.24201469123363495, "acc": 0.9099957942962646, "length": 3.9763224124908447, "mode": "train", "epoch": 30}
{"loss": 0.33776673674583435, "acc": 0.8747696876525879, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.2460668534040451, "acc": 0.902733325958252, "length": 3.9811618328094482, "mode": "train", "epoch": 31}
{"loss": 0.32816189527511597, "acc": 0.8683240413665771, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.22674082219600677, "acc": 0.9152937531471252, "length": 3.983355760574341, "mode": "train", "epoch": 32}
{"loss": 0.31090667843818665, "acc": 0.8802947402000427, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.2210579216480255, "acc": 0.914618730545044, "length": 3.9874908924102783, "mode": "train", "epoch": 33}
{"loss": 0.31254416704177856, "acc": 0.8766113519668579, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.21670210361480713, "acc": 0.9161317944526672, "length": 3.9976189136505127, "mode": "train", "epoch": 34}
{"loss": 0.3329342305660248, "acc": 0.8839778900146484, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.21819929778575897, "acc": 0.917931318283081, "length": 3.9955475330352783, "mode": "train", "epoch": 35}
{"loss": 0.3143979609012604, "acc": 0.8775323629379272, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.2125576287508011, "acc": 0.9158295392990112, "length": 3.997359275817871, "mode": "train", "epoch": 36}
{"loss": 0.3068203926086426, "acc": 0.8867403268814087, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.20563849806785583, "acc": 0.9166582822799683, "length": 3.9925167560577393, "mode": "train", "epoch": 37}
{"loss": 0.30724331736564636, "acc": 0.8830572366714478, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.20787933468818665, "acc": 0.9195684194564819, "length": 3.9989302158355713, "mode": "train", "epoch": 38}
{"loss": 0.3085513114929199, "acc": 0.8931860327720642, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.21052873134613037, "acc": 0.9172745943069458, "length": 3.994507074356079, "mode": "train", "epoch": 39}
{"loss": 0.31554198265075684, "acc": 0.8950276970863342, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.22635510563850403, "acc": 0.9154893159866333, "length": 3.99664306640625, "mode": "train", "epoch": 40}
{"loss": 0.2995632588863373, "acc": 0.8876612782478333, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.20827427506446838, "acc": 0.9182871580123901, "length": 3.9982573986053467, "mode": "train", "epoch": 41}
{"loss": 0.2967658042907715, "acc": 0.8830570578575134, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.2023027390241623, "acc": 0.9206563830375671, "length": 3.9960508346557617, "mode": "train", "epoch": 42}
{"loss": 0.30651360750198364, "acc": 0.8876611590385437, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.19412006437778473, "acc": 0.920072615146637, "length": 3.9945130348205566, "mode": "train", "epoch": 43}
{"loss": 0.30559033155441284, "acc": 0.883978009223938, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.19946672022342682, "acc": 0.923333466053009, "length": 3.991183280944824, "mode": "train", "epoch": 44}
{"loss": 0.27717721462249756, "acc": 0.9014735221862793, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.18857477605342865, "acc": 0.9309072494506836, "length": 3.9970123767852783, "mode": "train", "epoch": 45}
{"loss": 0.3137298822402954, "acc": 0.8977901339530945, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.18665987253189087, "acc": 0.9242352247238159, "length": 3.99981427192688, "mode": "train", "epoch": 46}
{"loss": 0.29620981216430664, "acc": 0.8968692421913147, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.18043263256549835, "acc": 0.9288683533668518, "length": 3.9996213912963867, "mode": "train", "epoch": 47}
{"loss": 0.31368887424468994, "acc": 0.8885819911956787, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.19092702865600586, "acc": 0.9251304864883423, "length": 3.9985880851745605, "mode": "train", "epoch": 48}
{"loss": 0.33053863048553467, "acc": 0.8968693614006042, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.1807214617729187, "acc": 0.9319391846656799, "length": 3.998213052749634, "mode": "train", "epoch": 49}
{"loss": 0.28809139132499695, "acc": 0.9023943543434143, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.18419016897678375, "acc": 0.9319709539413452, "length": 3.9999520778656006, "mode": "train", "epoch": 50}
{"loss": 0.29916301369667053, "acc": 0.9051566123962402, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.18492597341537476, "acc": 0.9310439229011536, "length": 3.999967336654663, "mode": "train", "epoch": 51}
{"loss": 0.29486843943595886, "acc": 0.903315007686615, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.17488211393356323, "acc": 0.9334166646003723, "length": 3.9957938194274902, "mode": "train", "epoch": 52}
{"loss": 0.29918980598449707, "acc": 0.8950278162956238, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.17583155632019043, "acc": 0.9322677850723267, "length": 3.999661684036255, "mode": "train", "epoch": 53}
{"loss": 0.33086520433425903, "acc": 0.8830570578575134, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.18828395009040833, "acc": 0.928516685962677, "length": 3.9989686012268066, "mode": "train", "epoch": 54}
{"loss": 0.3240034878253937, "acc": 0.8904236555099487, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.18190349638462067, "acc": 0.9285670518875122, "length": 3.9998791217803955, "mode": "train", "epoch": 55}
{"loss": 0.31198084354400635, "acc": 0.9014732241630554, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.18140925467014313, "acc": 0.9268612861633301, "length": 3.9963645935058594, "mode": "train", "epoch": 56}
{"loss": 0.31520068645477295, "acc": 0.903315007686615, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.182669535279274, "acc": 0.928555965423584, "length": 3.999310255050659, "mode": "train", "epoch": 57}
{"loss": 0.3095725476741791, "acc": 0.8959484696388245, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.1702478528022766, "acc": 0.9347382187843323, "length": 3.9989445209503174, "mode": "train", "epoch": 58}
{"loss": 0.3163280487060547, "acc": 0.892265260219574, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.18143858015537262, "acc": 0.9291893243789673, "length": 3.9999773502349854, "mode": "train", "epoch": 59}
{"loss": 0.3222878575325012, "acc": 0.8931860327720642, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.17106212675571442, "acc": 0.9336225986480713, "length": 3.994619846343994, "mode": "train", "epoch": 60}
{"loss": 0.3222211003303528, "acc": 0.8867403864860535, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.1676190197467804, "acc": 0.9344350099563599, "length": 3.9997806549072266, "mode": "train", "epoch": 61}
{"loss": 0.3000354468822479, "acc": 0.9005525708198547, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.16787688434123993, "acc": 0.9363012313842773, "length": 3.999985933303833, "mode": "train", "epoch": 62}
{"loss": 0.3223949074745178, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.17105743288993835, "acc": 0.931041955947876, "length": 3.9999210834503174, "mode": "train", "epoch": 63}
{"loss": 0.3221820592880249, "acc": 0.9060774445533752, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.16758392751216888, "acc": 0.9380292892456055, "length": 3.996487855911255, "mode": "train", "epoch": 64}
{"loss": 0.3284793496131897, "acc": 0.8977900743484497, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.16270960867404938, "acc": 0.9381564259529114, "length": 3.9999828338623047, "mode": "train", "epoch": 65}
{"loss": 0.33481916785240173, "acc": 0.8904235363006592, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.15845036506652832, "acc": 0.9409162402153015, "length": 3.9915733337402344, "mode": "train", "epoch": 66}
{"loss": 0.31935447454452515, "acc": 0.887661337852478, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.16719676554203033, "acc": 0.9390829801559448, "length": 3.9999582767486572, "mode": "train", "epoch": 67}
{"loss": 0.3077402710914612, "acc": 0.9069980978965759, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.16013291478157043, "acc": 0.9397029876708984, "length": 3.9999964237213135, "mode": "train", "epoch": 68}
{"loss": 0.30250734090805054, "acc": 0.901473343372345, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.16296237707138062, "acc": 0.9384657740592957, "length": 3.9999873638153076, "mode": "train", "epoch": 69}
{"loss": 0.3276645243167877, "acc": 0.8941068649291992, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.16689737141132355, "acc": 0.9393933415412903, "length": 3.999983787536621, "mode": "train", "epoch": 70}
{"loss": 0.27859199047088623, "acc": 0.908839762210846, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.16581805050373077, "acc": 0.9393938183784485, "length": 3.9999918937683105, "mode": "train", "epoch": 71}
{"loss": 0.3087628483772278, "acc": 0.9060773253440857, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.14768366515636444, "acc": 0.944031298160553, "length": 3.9999752044677734, "mode": "train", "epoch": 72}
{"loss": 0.3264293670654297, "acc": 0.9014732241630554, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.16356688737869263, "acc": 0.9415537714958191, "length": 3.9999139308929443, "mode": "train", "epoch": 73}
{"loss": 0.3037574291229248, "acc": 0.9042356610298157, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.15977756679058075, "acc": 0.9356787800788879, "length": 3.999910831451416, "mode": "train", "epoch": 74}
{"loss": 0.3068505823612213, "acc": 0.9042357802391052, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.15904222428798676, "acc": 0.9365783929824829, "length": 3.9992308616638184, "mode": "train", "epoch": 75}
{"loss": 0.31932687759399414, "acc": 0.9033148884773254, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.1574692726135254, "acc": 0.9405197501182556, "length": 3.9978439807891846, "mode": "train", "epoch": 76}
{"loss": 0.34951072931289673, "acc": 0.8867403268814087, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.14664296805858612, "acc": 0.9458861947059631, "length": 3.9999728202819824, "mode": "train", "epoch": 77}
{"loss": 0.34036117792129517, "acc": 0.8941066861152649, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.1465289145708084, "acc": 0.9415580034255981, "length": 3.9999899864196777, "mode": "train", "epoch": 78}
{"loss": 0.3111017942428589, "acc": 0.9023942351341248, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.15123088657855988, "acc": 0.9406304359436035, "length": 3.9999914169311523, "mode": "train", "epoch": 79}
{"loss": 0.2991463840007782, "acc": 0.9014732241630554, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.143745556473732, "acc": 0.9427953362464905, "length": 3.99999737739563, "mode": "train", "epoch": 80}
{"loss": 0.33098629117012024, "acc": 0.9042357802391052, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.1413213163614273, "acc": 0.9434137344360352, "length": 3.999999523162842, "mode": "train", "epoch": 81}
{"loss": 0.3329453468322754, "acc": 0.90239417552948, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.1441766768693924, "acc": 0.944959282875061, "length": 3.9999895095825195, "mode": "train", "epoch": 82}
{"loss": 0.3416420817375183, "acc": 0.8950276374816895, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.14065682888031006, "acc": 0.9499042630195618, "length": 3.999948263168335, "mode": "train", "epoch": 83}
{"loss": 0.2971329391002655, "acc": 0.90791916847229, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.13841694593429565, "acc": 0.9455780386924744, "length": 3.999998092651367, "mode": "train", "epoch": 84}
{"loss": 0.3593441843986511, "acc": 0.8941069841384888, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.14507390558719635, "acc": 0.94230717420578, "length": 3.995518445968628, "mode": "train", "epoch": 85}
{"loss": 0.31194007396698, "acc": 0.8987109065055847, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.15230953693389893, "acc": 0.9441359639167786, "length": 3.996417999267578, "mode": "train", "epoch": 86}
{"loss": 0.3669569194316864, "acc": 0.8876612782478333, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.14313039183616638, "acc": 0.9449598789215088, "length": 4.0, "mode": "train", "epoch": 87}
{"loss": 0.3088608384132385, "acc": 0.9051566123962402, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.13783328235149384, "acc": 0.9495980143547058, "length": 3.999999523162842, "mode": "train", "epoch": 88}
{"loss": 0.3090375065803528, "acc": 0.90239417552948, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.14013102650642395, "acc": 0.9455782175064087, "length": 4.0, "mode": "train", "epoch": 89}
{"loss": 0.29620981216430664, "acc": 0.9042356610298157, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.1329396516084671, "acc": 0.9486664533615112, "length": 3.999948263168335, "mode": "train", "epoch": 90}
{"loss": 0.314347505569458, "acc": 0.8977901339530945, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.15287335216999054, "acc": 0.9400122761726379, "length": 4.0, "mode": "train", "epoch": 91}
{"loss": 0.33699142932891846, "acc": 0.8867403268814087, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.15449370443820953, "acc": 0.9415583610534668, "length": 3.9999985694885254, "mode": "train", "epoch": 92}
{"loss": 0.2836378812789917, "acc": 0.9125230312347412, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.14887665212154388, "acc": 0.9452687501907349, "length": 3.9999918937683105, "mode": "train", "epoch": 93}
{"loss": 0.3473832607269287, "acc": 0.8922652006149292, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.1424611359834671, "acc": 0.9443413019180298, "length": 3.999999523162842, "mode": "train", "epoch": 94}
{"loss": 0.33115631341934204, "acc": 0.8996317982673645, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.14355036616325378, "acc": 0.9471243023872375, "length": 3.9999985694885254, "mode": "train", "epoch": 95}
{"loss": 0.34159737825393677, "acc": 0.9023942351341248, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.12596212327480316, "acc": 0.9533084630966187, "length": 3.999997854232788, "mode": "train", "epoch": 96}
{"loss": 0.332019567489624, "acc": 0.8968690633773804, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.12772446870803833, "acc": 0.9539270401000977, "length": 3.999999523162842, "mode": "train", "epoch": 97}
{"loss": 0.3125428855419159, "acc": 0.9079190492630005, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.1357192099094391, "acc": 0.9486700892448425, "length": 3.999992847442627, "mode": "train", "epoch": 98}
{"loss": 0.32183632254600525, "acc": 0.9033150672912598, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.13648147881031036, "acc": 0.9480517506599426, "length": 3.999993324279785, "mode": "train", "epoch": 99}
{"loss": 0.347395658493042, "acc": 0.9014732241630554, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.12789040803909302, "acc": 0.9502165913581848, "length": 4.0, "mode": "train", "epoch": 100}
{"loss": 0.3096616268157959, "acc": 0.8996317386627197, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.8194445371627808
